### PR TITLE
[17.0][MIG] stock_move_value_report: Migration to 17.0

### PR DESCRIPTION
**Pull request by vvrossem**
Based on #325 

From 16.0 to 17.0, in

- `stock.move.line`:
  - `reserved_qty` is removed. `quantity_product_uom` seems to replace it.
  - `qty_done` is removed. `quantity` seems to replace it.
- `stock.move`: the relation with `stock.scrap` changed from a one2many `scrap_ids` field  to a many2one `scrap_id` field
- `stock.scrap`: the relation with `stock.move` changed from a many2one `move_id` field  to a one2many `move_ids` field

I tried to adapt the code to reflect theses changes. An attentive review is appreciated :pray:

### CODE REVIEW

File: stock_move_value_report/report/stock_move_line_value_report.xml [hunk 1]
```diff
@@ -171,33 +173,39 @@
                     <t t-if="ml_from_scraps">
                         <t
                             t-set="scraps"
-                            t-value="move_lines_ids.mapped('move_id.scrap_ids').filtered(lambda x: x.move_id in ml_from_scraps.mapped('move_id'))"
+                            t-value="move_lines_ids.mapped('move_id.scrap_id').filtered(lambda x: x in ml_from_scraps.mapped('move_id.scrap_id'))"
```
Comment (yankinmax) [inline stock_move_value_report/report/stock_move_line_value_report.xml:176]: why do you map on `scrap_id` here:
`.filtered(lambda x: x in ml_from_scraps.mapped('move_id.scrap_id'))`?

**Review by yankinmax — APPROVED** [review]
LGTM, code review

**Review by Camille0907 — APPROVED** [review]
Migration code LGTM :)

**Pull request merged**